templates/Submissions/mdtest.php

```diff
@@ -1,5 +1,5 @@
 <div class="submissions index content">
-    <ul class="webroot/plots">
```

Comment:
Hmm, wasn't "webroot/" just being added in the first patch in this series?  It is much preferred if fixes are merged into the original patch, rather than being applied in a separate patch later in the series.

Reply:
Yes, it was, but I think this one was missing in the previous commit!

Reply:
As I wrote, for improved understanding of the patches, and to avoid landing partially-broken changes, it would be better to resubmit this PR with this fix included into the first patch where webroot was being modified, and also add longer explanations to the commit messages for each patch about what is being done, and why.

Consider it a mini tutorial explaining to PHP-dense people like me why this change is useful, and a record of how these plots are created/updated, so that there is some persistent record for reference in the future.  It may be obvious to _you_ how to do this now, but to me it is not, and in a year or five it will not be for you either.